ohara-manager/client/src/store/epics/broker/updateBrokerEpic.js

```diff
@@ -18,19 +18,22 @@ import { normalize } from 'normalizr';
 import { merge } from 'lodash';
 import { ofType } from 'redux-observable';
 import { defer, from } from 'rxjs';
-import { catchError, map, startWith, mergeMap } from 'rxjs/operators';
+import { catchError, map, startWith, mergeMap, tap } from 'rxjs/operators';
 
 import * as brokerApi from 'api/brokerApi';
 import * as actions from 'store/actions';
 import * as schema from 'store/schema';
 import { getId } from 'utils/object';
 import { LOG_LEVEL } from 'const';
 
-export const updateBroker$ = (values) => {
+export const updateBroker$ = (values, resolve) => {
   const brokerId = getId(values);
   return defer(() => brokerApi.update(values)).pipe(
     map((res) => res.data),
     map((data) => normalize(data, schema.broker)),
+    tap((data) => {
+      if (resolve) resolve(data);
+    }),
```

Comment:
```suggestion
    tap((data) => {
      if (resolve) resolve(data);
      return data;
    }),
```

應該要 return data，否則 next operator 接不到 data